metabox/metabox/metabox-provider/bin/gst_pipeline_test.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser
+import gi
+import sys
+import time
+gi.require_version('Gst', '1.0')
+gi.require_version('GLib', '2.0')
+from gi.repository import Gst        # noqa: E402
+from gi.repository import GLib       # noqa: E402
+
+
+def main():
+    parser = ArgumentParser(description='Simple GStreamer pipeline player')
+    parser.add_argument(
+        'PIPELINE',
+        help='Quoted GStreamer pipeline to launch')
+    parser.add_argument(
+        '-t', '--timeout',
+        type=int, required=True,
+        help='Timeout for running the pipeline')
+    args = parser.parse_args()
+    exit_code = 0
```

Comment:
This is never used except the end of the function, which would finish with None without the return statement yielding the same effect.